.pipelines/android_packaging.yml

```diff
@@ -0,0 +1,113 @@
+        inputs:
+          versionSpec: "11"
+          jdkArchitectureOption: "x64"
+          jdkSourceOption: "PreInstalled"
+
+      - template: templates/set-package-version-variable-step.yml
+        parameters:
+          IsReleaseBuild: ${{ parameters.IsReleaseBuild }}
+          PackageVersionVariableName: ORT_EXTENSIONS_AAR_VERSION
+
+      - script: |
+          python ./tools/gen_selectedops.py ./tools/android/package_ops.config
+        displayName: "Generate selected ops CMake file"
+
+      - script: |
+          bash ./tools/android.package.sh
+        displayName: Build onnxruntime-extensions aar package
+        workingDirectory: $(Build.sourcesdirectory)
+
+      - script: |
+          cd out/android/Release/java/androidtest/android
+
+          ./gradlew --no-daemon clean connectedDebugAndroidTest
+        displayName: Run extensions unit test with Android Emulator
+        workingDirectory: $(Build.sourcesdirectory)
+
+      - script: |
+          set -e -x
+          cd out/android/Release/java/androidtest/android
+
+          appcenter test run espresso \
+          --app "AI-Frameworks/Ort-Ext-Mobile-Android-Testapp" \
+          --devices $(app_center_android_test_devices) \
+          --app-path ./app/build/outputs/apk/debug/app-debug.apk \
+          --test-series "master" \
+          --locale "en_US" \
+          --build-dir ./app/build/outputs/apk/androidTest/debug \
+          --token $(app_center_api_token)
+        displayName: Run extensions Android unit test using App Center
+        workingDirectory: $(Build.sourcesdirectory)
+
+      - script: |
+          set -e -x
+
+          AAR_STAGING_DIR="$(Build.sourcesdirectory)/out/android/Release/java/build/android/outputs/aar"
+          LIBS_STAGING_DIR="$(Build.sourcesdirectory)/out/android/Release/java/build/android/libs"
+          POM_FILE_STAGING_DIR="$(Build.sourcesdirectory)/java/build/publications/maven"
+          ARTIFACTS_STAGING_DIR="$(Build.ArtifactStagingDirectory)"
+
+          PROJECT_NAME="onnxruntime-extensions"
+          PACKAGE_NAME="onnxruntime-mobile-extensions"
+          VERSION_FILE="$(Build.SourcesDirectory)/version.txt"
+          PROJECT_VERSION="$(cat "${VERSION_FILE}")"
+          cp ${LIBS_STAGING_DIR}/${PROJECT_NAME}-${PROJECT_VERSION}-javadoc.jar  ${ARTIFACTS_STAGING_DIR}
+          cp ${LIBS_STAGING_DIR}/${PROJECT_NAME}-${PROJECT_VERSION}-sources.jar  ${ARTIFACTS_STAGING_DIR}
+          cp ${AAR_STAGING_DIR}/${PROJECT_NAME}-release.aar                      ${ARTIFACTS_STAGING_DIR}
+          cp ${POM_FILE_STAGING_DIR}/pom-default.xml                             ${ARTIFACTS_STAGING_DIR}
+          mv ${ARTIFACTS_STAGING_DIR}/${PROJECT_NAME}-${PROJECT_VERSION}-javadoc.jar  ${ARTIFACTS_STAGING_DIR}/${PACKAGE_NAME}-${ORT_EXTENSIONS_AAR_VERSION}-javadoc.jar
+          mv ${ARTIFACTS_STAGING_DIR}/${PROJECT_NAME}-${PROJECT_VERSION}-sources.jar  ${ARTIFACTS_STAGING_DIR}/${PACKAGE_NAME}-${ORT_EXTENSIONS_AAR_VERSION}-sources.jar
```

Comment:
nit: instead of `cp <dir1>/<file> <dir2>; mv <dir2>/<file> <dir2>/<renamed file>` could just do `cp <dir1>/<file> <dir2>/<renamed file>`